### PR TITLE
Hide UiCollapsible content overflow only during animation

### PR DESCRIPTION
**Pull request by kilobyte2007**
Sometimes we need to have elements that are out of the collapsible scope (we have a color picker in our case) and it is hidden by the collapsible. But because we only need overflow hidden while animating we can assign it to Vue's animation classes.

### CODE REVIEW

File: src/UiCollapsible.vue [hunk 1]
```diff
@@ -217,8 +217,11 @@ $ui-collapsible-header-background-hover     : $md-grey-300 !default;
 
 .ui-collapsible__body-wrapper {
     max-height: 0;
-    overflow: hidden;
     transition: max-height 0.3s ease;
+
+    &.v-enter-active, &.v-leave-active {
```
Comment (JosephusPaye) [inline src/UiCollapsible.vue:222]:
```suggestion
    &.v-enter-active,
    &.v-leave-active {
```

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
Makes sense. Just a small change and we can merge...

**Review by JosephusPaye — APPROVED** [review]

**Comment by kilobyte2007** [comment]
Hey @JosephusPaye, can you please merge this? Thank you.

**Comment by JosephusPaye** [comment]
Thanks! Released in [v1.3.2](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.3.2)

**Pull request merged**